src/qdmi/Driver.cpp

```diff
@@ -0,0 +1,479 @@
+/*
+ * Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+ * Copyright (c) 2025 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#include "qdmi/Driver.hpp"
+
+#include "mqt_na_qdmi/device.h"
+
+#include <cassert>
+#include <cstddef>
+#include <cstring>
+#include <dlfcn.h>
```

Comment:
The seeds to be guarded by `#ifndef _WIN32` because it is not available on Windows.